transport/http/src/main/java/org/kaazing/gateway/transport/http/bridge/filter/HttpLoginSecurityFilter.java

```diff
@@ -182,7 +188,13 @@ private boolean loginMissingToken(NextFilter nextFilter,
             }
 
             loginContext.login();
+        } catch (AccessControlLoginException ace) {
+            if (loggerEnabled()) {
+                log("Login failed: ", ace);
+            }
 
+            writeResponse(HttpStatus.CLIENT_FORBIDDEN, nextFilter, session, httpRequest);
+            return false;
         } catch (LoginException le) {
             // Depending on the login modules configured, this could be
             // a very normal condition.
```

Comment:
Understanding the meaning of this comment goes hand-in-hand with my question about the need for a custom exception above.